src/components/navbar/Navbar.tsx

```diff
@@ -46,36 +90,54 @@ function Navbar() {
 
   return (
     <NavbarStyles>
-      <NavLink path={HOME_PAGE_ROUTE}>
+      <NavLink path={role === "freelancer" ? HOME_PAGE_ROUTE : CLIENT_HOME}>
         <img height="50px" alt="logo" src={logo} />
       </NavLink>
 
       {screenWidth < MOBILE_SCREEN_SIZE ? (
         ""
       ) : (
         <>
-          <NavLink path={SEARCH_WORK_ROUTE}>{t("SearchWork.title")}</NavLink>
-          <NavLink path={PROPOSALS_ROUTE}>{t("Proposals.title")}</NavLink>
-          <NavLink path={MY_JOBS_ROUTE}>{t("MyJobs.title")}</NavLink>
-          <NavLink path={DASHBOARD_ROUTE}>{t("Dashboard.title")}</NavLink>
+          {navLinks.map(({ title, to }) => (
+            <NavLink path={to}>{title}</NavLink>
+          ))}
         </>
       )}
 
       <NavBarButtons>
         <NavLink path={CHAT_ROUTE}>
           <NavBarButton icon={<MessageFilled />} title={t("Chat.title")} />
         </NavLink>
-        <NavLink path={SETTINGS_ROUTE}>
-          <NavBarButton icon={<SettingFilled />} title={t("Settings.title")} />
-        </NavLink>
-        <NavLink path={PROFILE_ROUTE}>
-          <NavBarButton icon={<UserOutlined />} title={t("Profile.title")} />
-        </NavLink>
-        <NavBarButton
-          icon={<LogoutOutlined />}
-          title={t("SignIn.signOut")}
-          onClick={signOut}
-        />
+        {role === "freelancer" ? (
+          <>
+            <NavLink path={SETTINGS_ROUTE}>
+              <NavBarButton
+                icon={<SettingFilled />}
+                title={t("Settings.title")}
+              />
+            </NavLink>
+            <NavLink path={PROFILE_ROUTE}>
+              <NavBarButton
+                icon={<UserOutlined />}
+                title={t("Profile.title")}
+              />
+            </NavLink>
+          </>
+        ) : (
+          <>
+            <NavLink path={CLIENT_PROFILE}>
+              <NavBarButton
+                icon={<UserOutlined />}
+                title={t("Profile.title")}
+              />
+            </NavLink>
+            <NavBarButton
+              icon={<LogoutOutlined />}
+              title={t("SignIn.signOut")}
+              onClick={signOut}
+            />
```

Comment:
Sign out button should be out of the conditional render so both freelancer and client can see it